src/console/c_cvars.cpp

```diff
@@ -1745,3 +1747,55 @@ CCMD (archivecvar)
 		}
 	}
 }
+
+void C_GrabCVarDefaults ()
+{
+	int lump, lastlump = 0;
+	int lumpversion, gamelastrunversion;
+	gamelastrunversion = atoi(LASTRUNVERSION);
+
+	while ((lump = Wads.FindLump("DEFCVARS", &lastlump)) != -1)
```

Comment:
This will work for any mod, not only IWAD. Is it intentional?

Reply:
Correct. It won't have a heck of a lot of use outside of IWADs though unless a player is really really new and starting with a fresh config file. The intent is indeed for IWADs but I don't see much reason to disallow it for regular mods. Sometimes, mods introduce people to GZDoom (such as Brutal Doom), so having a "way it's meant to be played" setup can't hurt, here.